### PR TITLE
Update registry.json - adding istanpool.info

### PR DESCRIPTION
**Pull request by eroler72**

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -3382,5 +3382,20 @@
...
+    "poolId": "bce4f15e81e2d2d9fe5c061f02f89ced9a56654a753f14d8409505bb",
+    "social": {
+      "facebook": "",
+      "twitter": "istanpool34",
+      "website": "https://istanpool.info",
+      "reddit": "",
+      "instagram": "",
+      "youtube": ""
+    },
+    "telegram": "istanpool.info",
```
Comment (Bez625) [inline registry.json:3397]: Hey I cant find this telegram handle. Please can you make sure this is a telegram handle of an individual user and not a group or channel? Thanks

**Review by Bez625 — CHANGES_REQUESTED** [review]

File: registry.json (hunk 1 above)
Comment (eroler72) [reply registry.json:3397]: Hi there, 
Thanks for reaching to me. 
do you mean this: https://t.me/istanpool or my telegram username is @istanpool

Comment (Bez625) [reply registry.json:3397]: `istanpool` is fine

Comment (eroler72) [reply registry.json:3397]: thanks, do I need to do anything else?

Comment (Bez625) [reply registry.json:3397]: Just change  the telegram handle to your telegram handle

Comment (eroler72) [reply registry.json:3397]: Sorry, I am not good at using GitHub, I hope did it correctly.

Comment (eroler72) [reply registry.json:3397]: the file was updated.

**Comment by kiriakos** [comment]
Closing this in favor of #350

**Pull request closed without merging**